text/src/swam/witx/parser/ImportContext.scala

```diff
@@ -0,0 +1,54 @@
+package swam.witx.parser
+
+import java.io.File
+import java.nio.file.{Files, NoSuchFileException, Path, Paths}
+
+import cats.effect._
+import fastparse.Parsed
+import swam.text.{ParserException, readFile}
+import swam.witx.WitxParser
+import swam.witx.unresolved.{AliasType, BaseWitxType, ModuleInterface}
+
+import scala.concurrent.ExecutionContext
+
+/**
+  *@author Javier Cabrera-Arteaga on 2020-03-18
+  */
+class ImportContext[F[_]](val includes: List[String])(implicit val F: Effect[F]) {
+
+  // TODO FIX !
+  implicit val cs = IO.contextShift(ExecutionContext.Implicits.global)
```

Comment:
Put an implicit `ContextShift` contraint. Look at how it is done for other classes (e.g. `Engine`).